include/cloysterhpc/functions.h

```diff
@@ -14,6 +14,7 @@
 namespace cloyster {
 // Globals
 extern bool dryRun;
+extern bool nodeless;
```

Comment:
`extern`?

We don't need this here.

I don't see this variable used in any other place than [src/services/shell.cpp](https://github.com/viniciusferrao/cloysterhpc/pull/56/files#diff-227eec29916d93b975b02ff8055bb5ad858040a4d0bbf04fa7d25392f626c667), perhaps this could be a variable passed to the `Shell` or `Cluster` constructor?

pinging @viniciusferrao

Reply:
For this, we need some refactoring. I'm closing it for now.